### PR TITLE
Adding query introspection on leaf node.

### PR DESCRIPTION
**Pull request by nihal95**
Adding query introspection on leaf node to get wqf and pos. Also modifying queryintro1 test method to test leaf node.

### CODE REVIEW

File: xapian-core/api/query.cc [hunk 1]
```diff
@@ -228,6 +228,12 @@ Query::get_subquery(size_t n) const
     return internal->get_subquery(n);
 }
 
+Xapian::termcount
+Query::get_leaf_wqf() const
+{
+	return internal->get_wqf();
```
Comment (ojwb) [inline xapian-core/api/query.cc]: Here (and also in the next file) the code in the function is over-indented - it should be indented by 4 spaces.

Set your editor to display tabs with width 8, to indent by 4 columns, and to tab-fill the indents and you should avoid this problem.

**Comment by ojwb** [comment]
Looking good.

One bit of admin I need to take care of before merging - are you OK with the patch licensing requirements?

https://trac.xapian.org/browser/git/xapian-core/HACKING#L1392

**Comment by nihal95** [comment]
Hey olly, I hold the copyrights to the code and dual-license them under each of the following licences:
GPL version 2 and all later versions and MIT/X license. Is this fine?

**Comment by ojwb** [comment]
Squashed into 4fb286495452f96c077fc31d85aaed95f1ab9ce5 and pushed.

Thanks!

**Pull request closed without merging**